### PR TITLE
Two new hp procurve show lldp commands

### PR DESCRIPTION
**Pull request by sliddjur**
##### ISSUE TYPE
 - New Template Pull Request]

##### COMPONENT
hp procurve show lldp info remote-device
hp procurve show lldp info remote-device detail

##### SUMMARY
I chose to use hp column names.
Detail command will match interfaces with format `2`, `1,3-5`, or `detail`. The `detail` keyword is not available on all hp procurves, but will match all interfaces with lldp information..

### CODE REVIEW

File: templates/hp_procurve_show_lldp_info_remote-device.textfsm [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Value Required LOCAL_PORT (\S+)
+Value CHASSI_ID ((?:[a-z0-9]{2}\s){5}[a-z0-9]{2}|\S+)
```
Comment (FragmentedPacket) [inline templates/hp_procurve_show_lldp_info_remote-device.textfsm:2]:
```suggestion
Value CHASSIS_ID ((?:[a-z0-9]{2}\s){5}[a-z0-9]{2}|\S+)
```

File: templates/hp_procurve_show_lldp_info_remote-device_detail.textfsm [hunk 2]
```diff
@@ -0,0 +1,52 @@
+Value Required LOCAL_PORT (\S+)
+Value CHASSI_TYPE (\S+)
```
Comment (FragmentedPacket) [inline templates/hp_procurve_show_lldp_info_remote-device_detail.textfsm:2]:
```suggestion
Value CHASSIS_TYPE (\S+)
Value CHASSIS_ID ([a-f0-9]{6}-[a-f0-9]{6}|(?:[a-z0-9]{2}\s){5}[a-z0-9]{2}|\S+)

```

File: templates/hp_procurve_show_lldp_info_remote-device.textfsm [hunk 3]
```diff
@@ -0,0 +1,20 @@
+Value Required LOCAL_PORT (\S+)
```
Comment (FragmentedPacket) [inline templates/hp_procurve_show_lldp_info_remote-device.textfsm:1]: I think we discussed this in NTC public Slack, but could you name the values that adhere to other LLDP templates? This helps users to be able to key off the same `keys` between different platforms.

**Comment by sliddjur** [comment]
@FragmentedPacket which one? There are more than one "show lldp xxx". And they all use different key values.

**Comment by FragmentedPacket** [comment]
I would go with the cisco_ios_show_lldp for the naming. We need to fix those at some point.

**Comment by sliddjur** [comment]
Ok updated version with field names more like cisco ios

File: templates/hp_procurve_show_lldp_info_remote-device_detail.textfsm [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  ^\s+PortDescr\s+:\s${PORT_DESCR}\s*$$
+  ^\s+Pvid\s+:\s${PVID}\s*$$
+  ^\s+System\sCapabilities\sSupported\s+:\s${SYSTEM_CAPABILITIES_SUPPORTED}\s*$$
+  ^\s+System\sCapabilities\sEnabled\s+:\s${SYSTEM_CAPABILITIES_ENABLED}\s*$$
+  ^\s+Remote\sManagement\sAddress\s*$$
+  ^\s+Type.*$$
+  ^\s+Address\s:\s${REMOTE_MANAGEMENT_ADDRESS}
+  ^\s+Poe\sPlus\sInformation\sDetail\s*$$
+  ^\s+Poe Device Type.*
+  ^\s+Power Source.*
```
Comment (FragmentedPacket) [inline templates/hp_procurve_show_lldp_info_remote-device_detail.textfsm:36]: One last thing, do you mind getting rid of static spaces and replacing them with `\s+`.

**Pull request closed without merging**